frontend/talentsearch/resources/js/components/communityExperienceForm/CommunityExperienceForm.tsx

```diff
@@ -0,0 +1,128 @@
+              description:
+                "Label displayed on Community Experience form for role input",
+            })}
+            name="role"
+            type="text"
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
+          />
+
+          <Input
+            id="organization"
+            label={intl.formatMessage({
+              defaultMessage: "Group / Organization / Community",
+              description:
+                "Label displayed on Community Experience form for organization input",
+            })}
+            name="organization"
+            type="text"
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
+          />
+
+          <Input
+            id="project"
+            label={intl.formatMessage({
+              defaultMessage: "Project / Product",
+              description:
+                "Label displayed on Community Experience form for project input",
+            })}
+            name="project"
+            type="text"
+            rules={{ required: intl.formatMessage(errorMessages.required) }}
+          />
+        </div>
+        <div>
+          <Checkbox
+            id="current-role"
+            label={intl.formatMessage({
+              defaultMessage: "I am currently active in this role",
+              description:
+                "Label displayed on Community Experience form for current role input",
+            })}
+            name="current-role"
+          />
+          <div data-h2-display="b(flex)">
+            <div data-h2-padding="b(right, l)">
+              <Input
+                id="start-date"
+                label={intl.formatMessage({
+                  defaultMessage: "Start Date",
+                  description:
+                    "Label displayed on Community Experience form for start date input",
+                })}
+                name="start-date"
+                type="date"
+                rules={{ required: intl.formatMessage(errorMessages.required) }}
+              />
+            </div>
+            <div>
+              {/* conditionally render the end-date based off the state attached to the checkbox input */}
+              {!isCurrent && (
+                <Input
+                  id="end-date"
+                  label={intl.formatMessage({
+                    defaultMessage: "End Date",
+                    description:
+                      "Label displayed on Community Experience form for end date input",
+                  })}
+                  name="end-date"
+                  type="date"
+                  rules={
+                    isCurrent
+                      ? {}
+                      : {
+                          required: intl.formatMessage(errorMessages.required),
+                          min: {
+                            value: startDate,
+                            message: intl.formatMessage(
+                              errorMessages.futureDate,
```

Comment:
The [`errorMessages.mustBeGreater`](https://github.com/GCTC-NTGC/gc-digital-talent/blob/50b5ad974d5ddab064eb13e52de256552031ecb0/frontend/common/src/messages/errorMessages.ts#L18-L22) error message is a better fit for this. You can set the value to `startDate` like this:

```js
message: intl.formatMessage(
  errorMessages.mustBeGreater,
  {
    value: startDate,
  },
),
```